BUILDING.rst

```diff
@@ -104,7 +104,7 @@ Install prerequisites
   - hipCUB, included for NVIDIA GPU targets, but required as an
     external dependency when building for AMD GPUs
   - roctracer-dev
-  - Linux kernel >= 3.5.0
+  - Linux kernel >= 3.6.0
```

Comment:
Is this version supposed to be bumped? Seems to do with linux kernel version and not hoomd version.

Reply:
You are correct, I will fix. Cause: `bump2version` is a simple pattern matcher and I missed this in my review.